doc/supply_demand.rst

```diff
@@ -169,6 +169,65 @@ Activated in Config from the `tes <https://github.com/PyPSA/pypsa-eur-sec/blob/3
 Thermal energy can be stored in large water pits associated with district heating systems and individual thermal energy storage (TES), i.e., small water tanks. Water tanks are modelled as `stores <https://pypsa.readthedocs.io/en/latest/components.html?highlight=distribution#store,  which are connected to heat demand buses through water charger/discharger links>`__.
 A thermal energy density of 46.8 kWh :math:`_{th}`/m3 is assumed, corresponding to a temperature difference of 40 K. The decay of thermal energy in the stores: 1- :math:`e^{-1/24τ}` is assumed to have a time constant  of  τ=180 days for central TES and  τ=3 days for individual TES, both modifiable through `tes_tau <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L229>`__ in config file. Charging and discharging efficiencies are 90% due to pipe losses.
 
+**Residential Heat Demand-Side Management (DSM)**
+
+Can be activated in the `config file <https://github.com/PyPSA/pypsa-eur/blob/master/config/config.default.yaml>`__ through the ``residential_heat_dsm`` option (default: false).
+
+Residential heat demand-side management allows electric heating systems to provide flexibility to the energy system by shifting heat demand within daily time windows while maintaining thermal comfort. This feature models the thermal mass of buildings as an energy storage capability, enabling heat pumps to operate more flexibly in response to electricity price signals and renewable generation variability.
+
+*Implementation approach*
+
+The implementation is based on the smartEn/DNV methodology for quantifying demand-side flexibility benefits in the European energy system. Building thermal mass is represented as energy stores connected to residential heat buses, with time-varying availability constraints that enforce consumption requirements within defined periods.
```

Comment:
Please link somewhere to the smartEn study and the precise location the assumptions are coming from, from within the study doc.